src/renderer/dx/DxFontRenderData.cpp

```diff
@@ -178,12 +178,46 @@ DxFontRenderData::DxFontRenderData(::Microsoft::WRL::ComPtr<IDWriteFactory1> dwr
 
 // Routine Description:
 // - Updates the font used for drawing
+// - This version is to be called when no features/axes were supplied, in which case we will
+//   use the ones we already know about (if any)
 // Arguments:
 // - desired - Information specifying the font that is requested
 // - actual - Filled with the nearest font actually chosen for drawing
 // - dpi - The DPI of the screen
 // Return Value:
 // - S_OK or relevant DirectX error
+[[nodiscard]] HRESULT DxFontRenderData::UpdateFont(const FontInfoDesired& desired, FontInfo& actual, const int dpi) noexcept
+{
+    try
+    {
+        _userLocaleName.clear();
+        _textFormatMap.clear();
+        _fontFaceMap.clear();
+        _boxDrawingEffect.Reset();
+
+        // Initialize the default font info and build everything from here.
+        _defaultFontInfo = DxFontInfo(desired.GetFaceName(),
+                                      desired.GetWeight(),
+                                      DWRITE_FONT_STYLE_NORMAL,
+                                      DWRITE_FONT_STRETCH_NORMAL);
+
+        _BuildFontRenderData(desired, actual, dpi);
+    }
+    CATCH_RETURN();
+
+    return S_OK;

```

Comment:
I don't like that this is exactly the same as the other `UpdateFont` only without the `_SetFeatures` and `_SetAxes` calls

Reply:
meh, I'm not sure how to make it better. A `const std::optional<std::unordered_map<std::wstring_view, uint32_t>>& features` is disgusting